### PR TITLE
Switch to new UI5 tooling 

### PR DESCRIPTION
**Pull request by WittmerJ**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+node_modules/
+coverage/
+dist/
+bower_components/
+src/
```
Comment (Michadelic) [inline .eslintignore]: why would you ignore the whole src and test folder in eslintignore? won't this avoid the code getting linted?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
-bower_components
-dist
+node_modules/
+dist/
+.idea/
+
+# only for testing, these resources should not be delivered but are easy to forget
+webapp/resources
+webapp/test-resources
+dist/webapp/resources
```
Comment (Michadelic) [inline .gitignore]: isn't it all dist?

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,11 @@
-node_modules
-bower_components
-dist
+node_modules/
+dist/
+.idea/
+
+# only for testing, these resources should not be delivered but are easy to forget
+webapp/resources
```
Comment (Michadelic) [inline .gitignore]: ignore whole webapp folder?

File: .npmignore [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
-.eslintrcn
+.eslintrc
+.eslintignore
+.idea
+webapp
+
+# only for testing, these resources should not be delivered but are easy to forget
+webapp/resources
+webapp/test-resources
+dist/webapp/resources
```
Comment (Michadelic) [inline .npmignore]: src and dist shall be included in npm module, but not test

File: webapp/index.html [hunk 5]
```diff
@@ -8,13 +8,14 @@
 	<title>UI5Lab Sample Browser</title>
 
 	<script id="sap-ui-bootstrap"
-			src="../../../resources/sap-ui-core.js"
-			data-sap-ui-libs="sap.m,sap.f"
+			src="/resources/sap-ui-core.js"
+			data-sap-ui-libs="sap.m, sap.f, sap.ui.layout"
```
Comment (Michadelic) [inline webapp/index.html]: minor: you only need to specify the libs here that are used in the html page, the other libs are loaded from the manifest:
sap.m should be enough if needed at all

File: webapp/manifest.json [hunk 6]
```diff
@@ -0,0 +1,150 @@
+{
+  "_version": "1.2.0",
```
Comment (Michadelic) [inline webapp/manifest.json]: please update to 1.12

File: webapp/manifest.json [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+	"i18n": "i18n/i18n.properties",
+	"title": "{{appTitle}}",
+	"description": "{{appDescription}}",
+	"applicationVersion": {
+	  "version": "1.0.0"
+	}
+  },
+
+  "sap.ui": {
+	"_version": "1.2.0",
```
Comment (Michadelic) [inline webapp/manifest.json]: please remove the _version in all subsections

File: webapp/manifest.json [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+	"rootView": {
+	  "viewName": "ui5lab.browser.view.App",
+	  "type": "XML",
+	  "id": "app",
+	  "async": true
+	},
+
+	"dependencies": {
+	  "minUI5Version": "1.36.0",
```
Comment (Michadelic) [inline webapp/manifest.json]: must be 1.46 since FlexibleColumnLayout is available sind 1.46

File: webapp/manifest.json [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+	  "libs": {
+		"sap.ui.core": {
+		  "minVersion": "1.36.0"
```
Comment (Michadelic) [inline webapp/manifest.json]: you can remove the minVersion here for all libs, it is not used anymore

File: .gitignore (hunk 3 above)
Comment (WittmerJ) [reply .gitignore]: if we ignore the whole webapp folder we won't have anything published to git

File: .eslintignore (hunk 1 above)
Comment (WittmerJ) [reply .eslintignore]: You are right in the point that it won't be linted. UI5Lab-Browser doesn't contain any src or test folder. We can delete it from the eslintignore

File: .npmignore (hunk 4 above)
Comment (WittmerJ) [reply .npmignore]: dist contains all data that is part of webapp (src). By including it in npm module we have it duplicate

**Pull request merged**